views/mart_web_app/grp_community_tracks.view.lkml

```diff
@@ -0,0 +1,115 @@
+# The name of this view in Looker is "Grp Community Tracks"
+view: grp_community_tracks {
+  sql_table_name: "MART_WEB_APP"."GRP_COMMUNITY_TRACKS"
+    ;;
+  # No primary key is defined for this view. In order to join this view in an Explore,
+  # define primary_key: yes on a dimension that has no repeated values.
+
+
+  dimension: _source_relation {
+    type: string
+    sql: ${TABLE}."_SOURCE_RELATION" ;;
+  }
+
+  dimension: category {
+    description: "The event's category."
+    type: string
+    sql: ${TABLE}."CATEGORY" ;;
+  }
+
+  dimension_group: event {
```

Comment:
why not call it `event_date`?

Reply:
yes, I fixed it, I missed catching it in the import of the view.